pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -377,6 +377,13 @@ stdenv.mkDerivation (finalAttrs: {
 
   outputs = [ "out" "dev" ] ++ (lib.optional (!buildLibsOnly) "man");
 
+
+  # Current `bpftool` can't handle BPF objects with compressed secrtions
+  # at least on i686-linux:
+  #   https://github.com/NixOS/nixpkgs/pull/272009#issuecomment-1846205807
```

Comment:
Upstream bpftool seems to be quite active, I'd be surprised if they'd not be interested in this being an issue. Can we open an issue at https://github.com/libbpf/bpftool?

Reply:
Sent question as https://lore.kernel.org/bpf/ZXcFRJVKbKxtEL5t@nz.home/T/#u as suggested by https://github.com/libbpf/libbpf#libbpf-and-general-bpf-usage-questions